kuma/payments/jinja2/payments/includes/payments-form.html

```diff
@@ -1,9 +1,3 @@
-{# Set URLs outside of translation block, to keep user's locale #}
```

Comment:
This section was added in PR #5052, and shouldn't be removed

Reply:
I have added it back